src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -775,8 +775,12 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <comment>{0} is the path to the project file. {1} is the restore duration.</comment>
   </data>
   <data name="Error_PackageDownload_OnlyExactVersionsAreAllowed" xml:space="preserve">
-    <value>'{0}' is not an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.</value>
-    <comment>0 - the version string that's not exact</comment>
+    <value>Package '{0} {1}' does not have an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.</value>
+    <comment>1 - the version string that's not exact. 0 - The package that does not have an exact version. Do not translate 'PackageDownload'.</comment>
```

Comment:
nit: the comments could be chronological. Not sure if it could confuse localization team at all.

Reply:
I agree. I have addressed it